go.mod

```diff
@@ -90,6 +90,8 @@ require (
 	k8s.io/klog/v2 v2.110.1
 )
 
+require github.com/nsf/jsondiff v0.0.0-20230430225905-43f6cf3098c1
```

Comment:
Why is this separate from the other direct dependencies?

Reply:
I executed, `go mod tidy` and it automatically added it there. I rearranged it manually in the latest commit.